src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,117 @@
+@import "./../utils/variables";
+
+%small-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
Create  a [mixin](https://sass-lang.com/documentation/at-rules/mixin/) this way. Or delete it, because it's not used anywhere.